src/runtime_src/hip/api/hip_event.cpp

```diff
@@ -51,7 +49,9 @@ static float hip_event_elapsed_time(hipEvent_t start, hipEvent_t stop)
   throw_invalid_value_if(!hip_ev_start, "dynamic_pointer_cast failed");
   auto hip_ev_stop = std::dynamic_pointer_cast<event>(command_cache.get(stop));
   throw_invalid_value_if(!hip_ev_stop, "dynamic_pointer_cast failed");
-  return hip_ev_start->elapsed_time(hip_ev_stop);
+  auto duration = hip_ev_stop->get_time() - hip_ev_start->get_time();
+  auto millis = std::chrono::duration_cast<std::chrono::milliseconds>(duration).count();
+  return millis;
```

Comment:
I believe there is an implicit type conversion from long to float here. Is this code clang-tidy clean?

Reply:
Thanks for the comment. I fixed it.